13-template.tex

```diff
@@ -47,16 +47,19 @@ \subsubsection{Functional Requirements}
 
 \subsubsection{Non-Functional Requirements}
 \begin{itemize}
-	\item Should be reasonably performant to run on a desktop/laptop
+	\item Should be reasonably performant to run on a desktop/laptop.
+	\item Should meet minimal security requirements for user recognition.
+	\item Should be smart enough to consistenly manage the whole house-system.  
 \end{itemize}
 
 \section{Analysis and domain model}
 
-The application will emulate a domotic controller connected to home devices.
-The home devices will also be emulated, each with its own logic.
-The user will be able to interact with the devices via the controller with a dashboard.
+The application will emulate a domotic environment, some smart devices and facilities monitoring emulation.
+Smart devices are common domestic devices as lights, electrical outlets, windows, doors and various sensors 
+for monitoring indoor temperature and air quality along with electrical absorption.
+The user will interact with the devices via the controller through a dashboard.
 
-The main difficulty will be modelling the communication between the controller and the different devices, which can vary in functioning.
+The main difficulty will be implementing a communication protocol for the controller and the devices, which can vary in functioning and tecchnical aspects. 
```

Comment:
```suggestion
The main difficulty will be implementing a communication protocol for the controller and the devices, which can vary in functioning and technical aspects. 
```

Reply:
@smagnaniuni next time you can just directly suggest the change like this. i find it more explicit.